### ISSUE TITLE
QueryPackages doesn't return SaveTo Property

### ISSUE THREAD

**Issue by tprivaterep** [Cr1TiKa7/My.Jdownloader.Api#6]
Using handler.DownloadsV2.QueryPackages(new LinkQueryRequest()); to query package doesn't return the SaveTo property
![image](https://user-images.githubusercontent.com/102672045/169577232-ba03ea64-cf82-4813-beb6-20bda9918bdc.png)

Example Console App
```
using My.JDownloader.Api;
using My.JDownloader.Api.Models.DownloadsV2.Request;
using My.JDownloader.Api.Models.DownloadsV2.Response;

JDownloaderHandler main = new JDownloaderHandler("xxxxxxxxxxxxxxxxxxxxxxx");
main.Connect("xxxxxxxxxxxxxxxxxxxxxxx", "xxxxxxxxxxxxxxxxxxxxxxx");

DeviceHandler handler = main.GetDeviceHandler(main.GetDevices().FirstOrDefault());

IEnumerable<DownloadLinkResponse> Links = handler.DownloadsV2.QueryLinks(new LinkQueryRequest());
IEnumerable<FilePackageResponse> Packages = handler.DownloadsV2.QueryPackages(new LinkQueryRequest());

```

**Comment by Cr1TiKa7** [comment]
Thanks for reporting! 
I've forgotton to add the 'SaveTo' property to the request object. That's the reason why it always was null.

Fixed in version 2.4.4

**Comment by tprivaterep** [comment]
Thanks bud for the fix :)
